RiverLoggerApi.Repository/DbModels/UserDbo.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace RiverLoggerApi.Repository.DbModels
+{
+    public class UserDbo
+    {
+        public Guid UserId { get; set; }
+
+        [Required]
+        public string Name { get; set; }
+
+        [Required]
+        public string LastName { get; set; }
+
+        [Required]
```

Comment:
I think you can add [EmailAddress] for validation but depends of the framework

Reply:
this should not be done in a dbo i will do it in the api level not at the repository level